src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +127,644 @@ export class JailAddComponent implements OnInit {
- a jail is possible only if the <b>allow_mount</b> and\
- <b>allow_mount_devfs</b> permissions are effective and\
- <b>enforce_statfs</b> is set to a value lower than <i>2</i>.'),
+      tooltip: T('The number of the <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=devfs&apropos=0&sektion=0&manpath=FreeBSD+11.1-RELEASE+and+Ports\
+"target="_blank"<b>devfs(8)<b> ruleset</a> to enforce when mounting \
+                 <b>devfs</b> in the jail. The default value of <i>0</i> \
+                 means no ruleset is enforced. Mounting <b>devfs</b> \
+                 inside a jail is only possible when the <b>allow_mount</b> \
+                 and <b>allow_mount_devfs</b> permissions are enabled \
+                 and <b>enforce_statfs</b> is set to a value lower than <i>2</i>.'),
     },
     {
       type: 'input',
       name: 'exec_start',
-      placeholder: T('exec_start'),
-      tooltip: T('Commands to run in the prison environment when a jail is\
- created. Example: <b>sh /etc/rc</b>. See <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=jail&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">jail(8)</a> for more details.'),
+      placeholder: T('exec.start'),
+      tooltip: T('Commands to run in the jail environment when the jail \
+                 is created. Example: <b>sh /etc/rc</b>. The pseudo-\
+                 parameters section of <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=jail&apropos=0&sektion=0&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+target="_blank">JAIL(8)</a> describes <b>exec.start</b> usage.'),
     },
     {
       type: 'input',
       name: 'exec_stop',
-      placeholder: T('exec_stop'),
-      tooltip: T('Commands to run in the prison environment before a jail\
- is removed and after any <b>exec_prestop</b> commands have completed.\
- Example: <i>sh /etc/rc.shutdown</i>.'),
+      placeholder: T('exec.stop'),
+      tooltip: T('Commands to run in the jail environment before the jail\
+                  is removed and after <b>exec.prestop</b> commands \
+                  have completed. Example: <i>sh /etc/rc.shutdown</i>.'),
     },
     {
       type: 'input',
       name: 'exec_prestart',
-      placeholder: T('exec_prestart'),
-      tooltip: T('List any commands to run in the system environment\
- before a jail is started.'),
+      placeholder: T('exec.prestart'),
+      tooltip: T('Commands to run in the system environment before a \
+                 before a jail is created.'),
     },
     {
       type: 'input',
       name: 'exec_poststart',
-      placeholder: T('exec_poststart'),
-      tooltip: T('List any commands to run in the system environment after\
- a jail is started and after any <b>exec_start</b> commands are finished.'),
-    }, {
+      placeholder: T('exec.poststart'),
+      tooltip: T('Commands to run in the system environment after a \
+                 jail is created and after any <b>exec.start</b> \
```

Comment:
The UI displays the relevant field name as "exec_start" - the text in bold needs to match with the field name exactly, either by adjusting the tooltip text or updating the "placeholder" value.